README.md

```diff
@@ -68,6 +68,36 @@ configured previously on page "New settings - Link checker robot".
 
 Once bot user is created "Bot user" line in status report should be showing "Good".
 
+## Disabling crawling of specific course categories
+
+This is achieved by configuring proper security roles in Moodle and assigning 
+these roles to the robot user on desired categories.
+
+Import role "Robot" from local/linkchecker_robot/roles/robot.xml on 
+Site administration ► Users ► Permissions ► Define roles ► Add a new role.
```

Comment:
need to add double lines break, look at the resultant markdown for what I mean:

https://github.com/central-queensland-uni/moodle-local_linkchecker_robot/blob/179bfcc0bc40adfe5df5b5486f461b9bd76761fe/README.md#disabling-crawling-of-specific-course-categories